core/src/main/java/digitalusage/rogers/com/actors/util/http/TopUpPurchasedClient.java

```diff
@@ -36,9 +34,10 @@
 
     private static final String backendError 				= "{0} [ response status = {1}, mw response = {2} ]";
 
-    private static String ucsClientId;
+    private static Map<String,String> ucsClientIdMapping = new HashMap<>();
```

Comment:
Lets make sure we add correct spacing/formatting
```suggestion
    private static Map<String, String> ucsClientIdMapping = new HashMap<>();
```